### DISCUSSION TITLE
How to make a best translation i18n of multiple style in nuxt or vue?

### DISCUSSION THREAD

**Discussion by rain-sc** [tailwindlabs/tailwindcss#11637] [Help]
locales
   en.json
   zh.json

`getLocale(){
  return locale.value // output: 'en' or 'zh'
}`

` <div
          class="flex"
          :class="getLocale==='en' ?
                text-[red]  font-bold  p-4 items-center :
                text-[blue] font-semibold p-2 items-start"
        >
          company name
        </div>
`

### it's better writing code to make a style?

**Comment by roma-marshall** [comment]
Hey @JoGlows, I will try to help you. Here's a small example.

First of all, you need to install the required packages `npm install vue-i18n`


**en.json**
```
{
  "hello": "Hello",
}
```
**zh.json**
```
{
  "hello": "你好",
}
```

**App.vue**
```
<template>
  <div>
    <p>{{ $t('hello') }}</p>
  </div>
</template>
```
